[conflict_resolver] Update test plan and README to test widgets (#6454)
aces/Loris: modules/conflict_resolver/README.md

with the data in LORIS, the `tools/recreate_conflicts.php` script
can be used to re-compare single and double data entry values
and repopulate the `conflicts_unresolved` table.

> The conflict resolver registers modules on the dashboard and the

Author: The README is the module spec. Users and admins are not the target audience.

> The conflict resolver registers modules on the dashboard and the

Reviewer: sure - developer-users and (sys)admins is what i meant. not end-users

> The conflict resolver registers modules on the dashboard and the

Author: The conflict resolver does not display anything. It registers widgets for the dashboard to use however it pleases.

> The conflict resolver registers modules on the dashboard and the

Reviewer 2: I agree with both. how about
```suggestion
The conflict resolver implements widgets for the `dashboard` and the
```
OR 
```suggestion
The conflict resolver offers widgets for the dashboard and the
```

> `candidate_profile` dashboard.

Reviewer 2: ```suggestion
`candidate_profile` modules.
```

I don't like calling it a dashboard to avoid confusion with THE dashboard which is a module

Author: It's emphatically designed to be a dashboard.
